package.json

```
@@ -10,8 +10,10 @@
   },
   "packageManager": "pnpm@8.1.1",
   "scripts": {
-    "dev": "nodemon --config nodemon.config.json -x 'turbo run build --filter=!fuel-gauge'",
-    "build": "turbo run build",
+    "prpa": "eslint . --ext .ts",
+    "dev": "nodemon --config nodemon.config.json -x 'pnpm build --filter=!fuel-gauge'",
+    "build": "pnpm build-forc-projects && turbo run build",
+    "build-forc-projects": "./scripts/build-script/build-all-with-cache.sh",
```

```suggestion
    "build:forc": "./scripts/build-script/build-all-with-cache.sh",
```
We tend to use the `build:some-thing` notation more consistently in the SDK.